test/unit/EmailService.js

```diff
@@ -41,7 +41,7 @@ describe("EmailService", () => {
         });
 
         const MathRandom = Math.random;
-        Math.random = () => 123456789987654321;
+        Math.random = () => 123456789987654321n;
```

Comment:
Why did you change this?

Reply:
Jest gave a warning because of that (after the bump). The normal int didn't have enough precision so the BigInt was needed